src/store/entitiesReducer.js

```diff
@@ -24,10 +24,10 @@ export const displayState = (data) => {
     return (<SystemState stateValue={ data }/>);
 };
 
-export const scoreProgress = (propName) => (data, _id, { display_performance_score: scoreObject }) => {
+export const scoreProgress = () => (data) => {
```

Comment:
As parameter `propName` is removed from here, could you also remove argument passed to this function?
Example - 
Here https://github.com/RedHatInsights/ros-frontend/blob/main/src/Routes/RosPage/RosPage.js#L39,  'cpu' argument is passed to the function.